File: labs/2023.03.03 - lab1/submit/task02_windows_testMonitoringSoftware/writeToFileCycle.c
#include <stdio.h>
#include <stdlib.h>
#include <Windows.h>

int main() {
    const char * const file = "writeToMeC.txt";
    while (1) {
        const int seconds = 3;
        printf("Sleeping for %i seconds\n", seconds); fflush(stdout);
        Sleep(seconds * 1000);

        printf("Writing to file: %s\n\n", file); fflush(stdout);
        FILE *fout = fopen(file, "w");
        fprintf(fout, "Hello\n");
        fclose(fout);
    }

    return 0;
}

File: labs/2023.03.03 - lab1/submit/task02_windows_testMonitoringSoftware/writeToFileCycle.py
import time

file = 'writeToMe.txt'
while True:
    seconds = 3
    print(f"Sleeping for {seconds}")
    time.sleep(seconds)

    with open(file, "w") as fout:
        print(f"Writing to file: {file}\n")
        fout.write("Hello \n")

